scripts/zones/Bastok_Markets/npcs/Yafafa.lua

```diff
@@ -18,7 +18,7 @@ entity.onTrigger = function(player, npc)
         {
             xi.item.BULB_OF_MHAURA_GARLIC,      83,
             xi.item.YAGUDO_CHERRY,              45,
-            xi.item.DHALMEL_MEAT,              249,
+            xi.item.SLICE_OF_DHALMEL_MEAT,              249,
```

Comment:
Please re-align this.

The rest of the changes -> :cheffkiss:

Reply:
Sorry about that, All done